File: VB/06_Properties/SetPropertiesForTemplate.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation
Imports Spire.Presentation.Drawing
Imports System.IO

Namespace SetPropertiesForTemplate
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'String for .pptx file 
			Dim pptxResult As String = "Output.pptx"

			'String for .odp file 
			Dim odpResult As String = "Output.odp"

			'String for .ppt file 
			Dim pptResult As String = "Output.ppt"

			'Create the .pptx template
			SetPropertiesForTemplate(pptxResult, FileFormat.Pptx2013)

			'Create the .odp template
			SetPropertiesForTemplate(odpResult, FileFormat.ODP)

			'Create the .ppt template
			SetPropertiesForTemplate(pptResult, FileFormat.PPT)

			'Launching the .pptx file.
			Viewer(pptxResult)
		End Sub
		Private Shared Sub SetPropertiesForTemplate(ByVal filePath As String, ByVal fileFormat As FileFormat)
			'Create a document
			Dim presentation As New Presentation()

			'Set the DocumentProperty 
			presentation.DocumentProperty.Application = "Spire.Presentation"
			presentation.DocumentProperty.Author = "E-iceblue"
			presentation.DocumentProperty.Company = "E-iceblue Co., Ltd."
			presentation.DocumentProperty.Keywords = "Demo File"
			presentation.DocumentProperty.Comments = "This file is used to test Spire.Presentation."
			presentation.DocumentProperty.Category = "Demo"
			presentation.DocumentProperty.Title = "This is a demo file."
			presentation.DocumentProperty.Subject = "Test"

			'Save to template file
			presentation.SaveToFile(filePath, fileFormat)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/06_Properties/SetPropertiesForTemplate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;
using Spire.Presentation.Drawing;
using System.IO;

namespace SetPropertiesForTemplate
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }
        private void btnRun_Click(object sender, EventArgs e)
        {
            //String for .pptx file 
            String pptxResult = "Output.pptx";

            //String for .odp file 
            String odpResult = "Output.odp";

            //String for .ppt file 
            String pptResult = "Output.ppt";

            //Create the .pptx template
            SetPropertiesForTemplate(pptxResult, FileFormat.Pptx2013);

            //Create the .odp template
            SetPropertiesForTemplate(odpResult, FileFormat.ODP);

            //Create the .ppt template
            SetPropertiesForTemplate(pptResult, FileFormat.PPT);

            //Launching the .pptx file.
            Viewer(pptxResult);
        }
        private static void SetPropertiesForTemplate(string filePath, FileFormat fileFormat)
        {
            //Create a document
            Presentation presentation = new Presentation();

            //Set the DocumentProperty 
            presentation.DocumentProperty.Application = "Spire.Presentation";
            presentation.DocumentProperty.Author = "E-iceblue";
            presentation.DocumentProperty.Company = "E-iceblue Co., Ltd.";
            presentation.DocumentProperty.Keywords = "Demo File";
            presentation.DocumentProperty.Comments = "This file is used to test Spire.Presentation.";
            presentation.DocumentProperty.Category = "Demo";
            presentation.DocumentProperty.Title = "This is a demo file.";
            presentation.DocumentProperty.Subject = "Test";

            //Save to template file
            presentation.SaveToFile(filePath, fileFormat);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}